darwin/dataset/remote_dataset_v2.py

```diff
@@ -889,7 +889,9 @@ def _get_remote_files_that_require_legacy_scaling(self) -> List[Path]:
             A list of full remote paths of dataset items that require NifTI annotations to be scaled
         """
         remote_files_that_require_legacy_scaling = []
-        remote_files = self.fetch_remote_files()
+        remote_files = self.fetch_remote_files(
+            filters={"statuses": ["new", "annotate", "review", "complete"]}
+        )
```

Comment:
These are stage types not item statuses right? PR description mentions `uploading`, `processing`, `error`, and `archived`; is that what is meant here?

Reply:
This is the `status` that `list_items` returns. In the backend, this is [`GeneralStatus`](https://github.com/v7labs/darwin-backend/blob/a971c7409e4ed51bfc78b3419c2a0b1ee08a757b/lib/graphotate_web/schemas/datasets_v2/common.ex#L194)

Reply:
Gotcha thanks, I guess it would be nice to have types/objects that can be used for these but maybe a separate piece of work